pkgs/development/libraries/hyperscan/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchurl
+, pkgconfig
+, boost
+, cmake
+, python
+, ragel
+}:
+
+let
+
+in
+stdenv.mkDerivation rec {
+  name = "hyperscan-${version}";
+  version = "4.6.0";
+
+  src = fetchurl {
+    url = "https://github.com/intel/hyperscan/archive/v${version}.tar.gz";
+    sha256 = "05d8zpjcm13lldkzcxafkabllzgv10nrhgfazsrag9l2bqpgryqd";
+  };
+
+  nativeBuildInputs = [
+    boost
+    cmake
+    pkgconfig
+    python
+    ragel
+  ];
+
+  enableParallelBuilding = true;
+
+  cmakeFlags = [
+    "-DBUILD_AVX512=ON"
+    "-DBUILD_SHARED_LIBS=ON"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://01.org/hyperscan;
+    description = "A high-performance multiple regex matching library";
+    license = [ licenses.bsd3 licenses.bsd2 licenses.boost ];
```

Comment:
From https://01.org/hyperscan it seems that hyperscan is only under BSD 3-Clause, and not under BSD 2-Clause or Boost licenses.

Reply:
See https://github.com/intel/hyperscan/blob/master/LICENSE

Reply:
It's common for big companies to list all dependencies' licenses in the license file. The package itself is still under BSD 3-Clause.

Reply:
https://github.com/intel/hyperscan/blob/d2e5089dc33c3f9d762898eefece67fe5ab323ea/include/boost-patched/graph/dominator_tree.hpp#L4

https://github.com/intel/hyperscan/blob/a00bd3167cda4d963ed71220cbbb703c06955f2d/src/crc32.c#L47

--> https://opensource.org/licenses/bsd-license.html